content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Manual Rotation of CA Certificates
+min-kubernetes-server-version: v1.13
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate the certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see [Authenticating](/docs/reference/access-authn-authz/authentication).
+For more information about best practices for CA certificates, see [Single root CA](docs/setup/best-practices/certificates/#single-root-ca).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+## Choose an approach
+There are two ways to rotate or replace CA certificates.
+
+## Replace the cluster (recommended) {#replace-the-cluster}
+
+This is the safest approach to rotate or replace the CA certificates.
+
+1. Create a new cluster with new CA certificates.
+
+2. Migrate the workloads to the new cluster.
+
+3. Remove or destroy the old cluster.
+
+## Rotate the CA certificates manually
+
+{{< caution >}}
+Make sure to back up your certificate directory along with configuration files (`.conf`) and any other necessary files.
+
+Manually rotating the CA works by timing when a Kubernetes cluster knows about new certificates. Do not restart any pods or update any component in the cluster while manually rotating the CA.
```

Comment:
> Do not restart any pods or update any component in the cluster while manually rotating the CA.

Since the idea for this guide is a zero-downtime rollout, regular pod restarts should be handled just fine